composer.json

```diff
@@ -34,7 +34,9 @@
         "drupal/autologout": "^1",
         "drupal/checklistapi": "^2.0",
         "drupal/collapsiblock": "^3",
+        "drupal/config_ignore": "^2.3",
         "drupal/config_rewrite": "1.3",
+        "drupal/config_split": "^1.7",
```

Comment:
@chandan-singh7929 As I remember, you mentioned config_split is required by blt. Can you confirm, what issues you were facing, when we don't include this ?

Reply:
It will show error message like "config_split module not found" as error but will not hold the script execution.

Reply:
@chandan-singh7929 I think we can skip this module and should not add in our project. Because this error is not due to our ACMS project, it's due to blt setup. So, In blt project itself, they can run the composer command to include this module and get rid of this error.

Reply:
make sense, let me update it.

Reply:
@vishalkhode1 - Its done, please have look.